lib/src/crypto.dart

```diff
@@ -2,8 +2,8 @@ import 'dart:convert';
 import 'package:crypto/crypto.dart';
 
 /// Generates an MD5 hash of the input string with the given salt
-String getMD5Hash(String input, String salt) {
-  final saltedInput = '$salt$input';
+String getMD5Hash(String input, {String? salt}) {
```

Comment:
made `salt` optional - this is the dart style for optional positional parameters